website/docs/en/plugins/rspack/html-rspack-plugin.mdx

```diff
@@ -85,6 +85,15 @@ Does not escape the content within the interpolation:
 <p>.</p>
 ```
 
+#### Control Statements
+
+Use the `for in` statement to implement list traversal and the `if` statement to implement conditional judgment:
+
+```html title="ejs"
+<% for tag in htmlRspackPlugin.tags.headTags { %> <% if tag.tagName=="script" {
```

Comment:
This code seems to be formatted incorrectly.

Reply:
seems to be formatted by prettier in git hooks, use txt code type instead